whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -875,14 +883,18 @@ class PostgreSQLComponent {
         }
     }
 
-    private PreparedStatement rigInsertStatement(PreparedStatement insert, Document doc, String changedIn, String changedBy, String collection, boolean deleted) {
+    private PreparedStatement rigInsertStatement(PreparedStatement insert, Document doc, Date timestamp, String changedIn, String changedBy, String collection, boolean deleted) {
```

Comment:
By doing timestamp.getTime() a bunch of times instead of just once and then passing the timestamp as a parameter (as was done before), we will end up in situations where these timestamps are no longer identical (time will have moved/changed from one call to getTime() to the next).

Reply:
No, `timestamp.getTime()` returns the same value every time.

Reply:
Aah, I looked to quickly! You're right of course! :)